isis/src/qisis/objs/Directory/ImportMapTemplateWorkOrder.cpp

```diff
@@ -107,47 +107,25 @@ namespace Isis {
    *
    * @return bool Returns true if we have at least one template file name.
    */
-  bool ImportTemplateWorkOrder::setupExecution() {
-    WorkOrder::setupExecution();
-
-    QString itemType;
-    QString filterText =
-        "Please select a file type;; Maps (*.def *.map *.pvl);; Registrations (*.def *.pvl)";
-
-    // If clicked "File"->"Import"->"Import Templates"
-    if (internalData().isEmpty()) {
-      itemType = "Templates";
-    }
-    // If clicked "Import Templates" from under "Maps" or "Registrations" ProjectItems rightclicks
-    else {
-      itemType = internalData().at(0);
+  bool ImportMapTemplateWorkOrder::setupExecution() {
 
-      if (itemType == "Maps") { filterText = "Maps (*.def *.map *.pvl)"; }
-      else if (itemType == "Registrations") { filterText = "Registrations (*.def *.pvl)"; }
-
-      internalData().clear();
-    }
+    WorkOrder::setupExecution();
 
-    QString selectedFilter;
     QStringList templateFileNames;
 
     templateFileNames = QFileDialog::getOpenFileNames(
         qobject_cast<QWidget *>(parent()),
-        "Import " + itemType,
+        "Import Map Templates",
         QString(),
-        filterText,
-        &selectedFilter);
+        "Maps (*.map);; All Files (*)");
```

Comment:
Is it valid to have a mapfile with a filename like `map.pvl`? If not, this makes sense.

Reply:
It is valid. I guess I am not sure of the concern here...? (If it is about the "Maps (*.map)..." line, this is simply a filter of the qfiledialog box. They can certainly get to .pvl files by choosing the "All Files" filter.)

Reply:
Then, no problem! I just wasn't picturing this correctly. Thanks!